cristin-commons/src/main/java/no/unit/nva/cristin/common/client/IClientProvider.java

```diff
@@ -0,0 +1,7 @@
+package no.unit.nva.cristin.common.client;
+
+public interface IClientProvider<T> {
```

Comment:
Let's not write C#. I don't think marking interfaces as interfaces is wise — in fact, it is something we explicitly don't want since this is code that is designed to express the business need, not the implementation specifics (I consider the fact that it is an interface to be one)

Reply:
I am aware that this is C# naming, but I never have had a good explanation why it is frown upon in Java.
Do you have one?